_data/SerialNo.yml

```diff
@@ -0,0 +1,23 @@
+- i: 1
```

Comment:
What is this/how is this file being used?

Reply:
I was trying to do something then figured out another way.
How I have deleted this extra file